CSharpMath/Atoms/Factories/MathListBuilder.cs

```diff
@@ -706,7 +706,7 @@ internal IMathList BuildInternal(bool oneCharOnly)
             }
           case MathAtomType.LargeOperator: {
               var op = (LargeOperator)atom;
-              var command = MathAtoms.LatexSymbolNameForAtom(op);
+              var command = MathAtoms.LatexSymbolNameForAtom(new LargeOperator(op.Nucleus, null));

```

Comment:
I suppose it's a quick hot-fix to avoid annoying exceptions until a final decision about how to handle it properly will be made.

Reply:
I understand, but what i cannot understand is that why mathatoms shouldn't have better identifier unlike nucleus? On my PR i've added command name to the mathatom and was not able to understand whats the problem with that ?

Reply:
Well, because it wasn't just identifier. How ID can be empty by default ? :-) 

Take a look on a latest comment for your PR:

> Right now this pull request makes the LaTeX-generating code a weird superposition between atoms generating their own LaTeX and a specialised type generating LaTeX from MathAtoms.

Reply:
I.e. currently there are options:

0. ignore "scripted" info when LaTeX is requested. [ current implementation ] 
1. provide command name (What was done in your PR)
2. override Equals for MathAtom
3. pass a custom IEqualityComparer<MathAtom>
4. get rid of AliasDictionary here: let MathAtom generate LaTeX by itself.
5. add another sub-dictionaries into AliasDictionary

Sorry, right now I don't have time to describe advantages and disadvantages of each approaches (on my opinion, of course). But I'd say current version of your PR doesn't bring improvements.

Also, maybe it makes sense to move code that improve generated LaTeX but doesn't change the way it is achieved into a separate PR.